docs/source/scenarios/supply_chain.rst

```diff
@@ -0,0 +1,225 @@
+  
+  * Transportation cost & delay penalty
+  
+  * Storage capacity & cost
+  
+  * SKUs' selling prices
+
+* Global settings
+  
+  * Reward discount
+  
+  * Initial balance
+
+
+Topologies
+----------
+
+Toy Topologie
+^^^^^^^^^^^^^
+
+In order to give users a more intuitive explanation of this scenario, MARO provides a toy topology, which can be called a "tiny supply chain world". The configuration file path is ``maro/simulator/scenarios/supply_chain/topologies/sample/config.yml``.
+
+The world in this topology is a 20*20 grid. There are three types of SKUs, named ``sku1``, ``sku2``, and ``sku3``. ``sku1`` requires ``sku3`` as raw material, while ``sku2`` and ``sku3`` do not need any material as input.
+
+Details of facilities:
+
+* ``Supplier_001``: located at [0, 0]
+  
+  * Produces ``sku3``.
+  
+  * Has small storage, a consumer, and a distribution.
+
+* ``Supplier_002``: located at [3, 3]
+  
+  * Produces ``sku1``, therefore need to consume ``sku3`` produced by other facilities as material.
+  
+  * Has small storage, a consumer,  and a distribution.
+
+* ``Warehouse_001``: located at [6, 6]
+  
+  * Could store all kinds of SKUs.
+  
+  * Has huge storage, a consumer, and a distribution.
+
+* ``Retailer_001``: located at [10, 18]
+  
+  * Could sell all kinds of SKUs.
+  
+  * Has medium storage and a consumer.
+
+The upstream-downstream dependency of this topology is simple, ``Supplier_001`` performs an upstream of ``Supplier_002`` (provides ``sku3`` as material for ``Supplier_002`` to produce ``sku1``), and both ``Supplier_001`` and ``Supplier_002`` are upstream facilities of ``Warehouse_001`` and ``Retailer_001``.
+
+Real Topologies ?
+^^^^^^^^^^^^^^^^^
+
+Naive Baseline
+^^^^^^^^^^^^^^
+
+TBD
+
+Quick Start
+-----------
+
+To start a simulation in the Supply Chain scenario, no extra data processing is needed. You can just go through the topology's configuration to build a general idea of this scenario, and then initialize an environment and enjoy your exploration in this scenario.
+
+Environment Interface
+^^^^^^^^^^^^^^^^^^^^^
+
+**Demand**
+
+As introduced above, the fundamental basis that affects the operation of the entire system is the prediction of market demand in the future. From the system's perspective, the change in demand is determined by the outside and cannot be controlled by itself. MARO has a built-in demand predictor, which samples future demand from a Gamma distribution (``SellerUnit``). MARO also provides an interface that accepts user-customized demand simulators, so that users could use their own demand prediction to guide the supply chain system (``OuterSellerUnit``).
+
+**System Status**
+
+In order to get known of the system status, we could find the information we need from the following two interfaces:
+
+* ``env.summary`` contains all static information about the system. For example, SKU list, unit settings, facilities settings, spatial distribution of facilities, etc..

```

Comment:
list them all?

How about use a code block to show the structure of the data structure?